plugin.py

```diff
@@ -1,180 +1,80 @@
 import os
 import hashlib
-import shutil
-import threading
-import urllib.request
 import sublime
 
-from sublime_lib import ActivityIndicator
+from urllib.request import urlretrieve
 
-from LSP.plugin.core.handlers import LanguageHandler
-from LSP.plugin.core.settings import ClientConfig, read_client_config
+from LSP.plugin.core.sessions import AbstractPlugin
+from LSP.plugin.core.sessions import register_plugin
+from LSP.plugin.core.sessions import unregister_plugin
```

Comment:
You should use `from LSP.plugin import ...`, I'll try to keep `plugin/__init__.py` stable (or deprecate things in advance in that file)